svgdigitizer/entrypoint.py

```diff
@@ -588,6 +591,57 @@ def _create_svg(svg, img, template_file, linked):
         drawing.save(pretty=True)
 
 
+@click.command()
+@click.argument("pdf")
+def get_doi(pdf):
```

Comment:
So basically this becomes obsolete with the PDF class

Reply:
This allows the user to quickly get the DOI of a publication via cli.

Reply:
ok, I did not realize that you added this as CLI command